lib/delayed_job_worker_pool/worker_group_options.rb

```diff
@@ -0,0 +1,22 @@
+module DelayedJobWorkerPool
+  class WorkerGroupOptions
+    DJ_SETTINGS = %i[
+      queues
+      min_priority
+      max_priority
+      sleep_delay
+      read_ahead
+    ].freeze
+    GROUP_SETTINGS = %i[workers].freeze
+    ALL_SETTINGS = (DJ_SETTINGS + GROUP_SETTINGS).freeze
+
+    ALL_SETTINGS.each { |setting| attr_accessor setting }
```

Comment:
Minor but we can shorten this up a bit.

```suggestion
    attr_accessor *ALL_SETTINGS
```